test/src/e2e.dynval/dv.n'.test.ts

```diff
@@ -134,4 +134,267 @@ describe("Dynamic Validator N -> N'", function() {
+        //   betty should be a validator in the second term.
+        const allDynNodes = withNodes(this, {
+            promiseExpect,
+            overrideParams: {
+                termSeconds: TERM_SECONDS
+            },
+            validators: [
+                { signer: alice, delegation: 5000, deposit: 100000 },
+                { signer: betty, delegation: 5000, deposit: 100 },
+                ...otherDynValidators.map((validator, index) => ({
+                    signer: validator,
+                    delegation: 5000 - index,
+                    deposit: 100000
+                }))
+            ],
+            onBeforeEnable: async allDynNodes => {
+                const aliceNode = allDynNodes[0];
+                // Kill the alice node first to make alice not to participate in the term 1.
+                await aliceNode.clean();
+            }
+        });
+
+        it("Alice should get out of the committee and Betty should be included in the committee", async function() {
+            const margin = 1.1;
+            const [_aliceNode, bettyNode, ...otherDynNodes] = allDynNodes;
+            {
+                const authors = (await stake.getPossibleAuthors(
+                    otherDynNodes[0].sdk
+                ))!.map(author => author.toString());
+                expect(authors).to.includes(alice.platformAddress.toString());
+                expect(authors).not.to.includes(
+                    betty.platformAddress.toString()
+                );
+                expect(authors.length).to.be.equals(8);
+            }
+
+            {
+                const tx = stake
+                    .createSelfNominateTransaction(bettyNode.sdk, 100000, "")
+                    .sign({
+                        secret: betty.privateKey,
+                        seq: await bettyNode.sdk.rpc.chain.getSeq(
+                            betty.platformAddress
+                        ),
+                        fee: 10
+                    });
+
+                await bettyNode.sdk.rpc.chain.sendSignedTransaction(tx);
+            }
+
+            await otherDynNodes[0].waitForTermChange(2, TERM_SECONDS * margin);
+
+            const blockNumber = await otherDynNodes[0].sdk.rpc.chain.getBestBlockNumber();
+            const termMetadata = await stake.getTermMetadata(
+                otherDynNodes[0].sdk,
+                blockNumber
+            );
+
+            {
+                const authors = (await stake.getPossibleAuthors(
+                    otherDynNodes[0].sdk
+                ))!.map(author => author.toString());
+                expect(authors).not.to.includes(
+                    alice.platformAddress.toString()
+                );
+                expect(authors).to.includes(betty.platformAddress.toString());
+                expect(authors.length).to.be.equals(8);
+            }
+
+            expect(termMetadata).not.to.be.null;
```

Comment:
I think this is not a place to check this. This test should only test if the validator change is working well.